Pollo/Supporting/BannerController.swift

```diff
@@ -7,15 +7,177 @@
 //
 
 import Foundation
+import FutureNova
+import UIKit
 import NotificationBannerSwift
 
+protocol PollingViewControllerDelegate: class {
+    func pollsDateViewControllerWasPopped(for userRole: UserRole)
+}
+
+class PollingViewController: UIViewController {
```

Comment:
can we move this to a `PollingViewController.swift` file since its kind of large